bazel/variables.bzl

```diff
@@ -1,4 +1,37 @@
-DEFAULT_COPTS = [
+# Those are the build flags used to build MalDocA (maldoca/*).
+# The flags are not applied to build third party dependencies.
+
```

Comment:
nit: delete this empty line.

Reply:
the empty line is deliberate. it indicates that the comment is a file wide comment and does not only apply to the next flag (i.e. DEFAULT_MALDOCA_WIN_COPTS)

Reply:
I see, thanks